.github/ISSUE_TEMPLATE/Repodiving_Request.md

```diff
@@ -0,0 +1,16 @@
+---
+name: Repodiving Request
+about: Ticket for Repodiving Request
+title: Repodiving Request
+labels: helpdesk
+assignees: ''
+
+---
+# Repodiving Request
+
+## What type of repodive would you like? (E.g., Cocomo or Repolinter or Gource-logs or Code.json-autogenerator)
+
+- [ ] Cocomo
```

Comment:
It would be helpful to add descriptions with more information on each repodive service